Core/Inc/drivers/spi/spi_obc.h

```diff
@@ -0,0 +1,17 @@
+/*
+ * spi_obc.h
+ *
+ *  Created on: Jul 21, 2022
+ *      Author: jenish
+ */
+
+#ifndef INC_DRIVERS_SPI_SPI_OBC_H_
+#define INC_DRIVERS_SPI_SPI_OBC_H_
+
+#include <iris_system.h>
+
+void obc_spi_transmit(uint8_t *tx_data, uint16_t data_length);
```

Comment:
The HAL functions in these wrappers return a HAL_StatusTypeDef - we should use them for error checkin

Reply:
Agreed